db/migrate/20240309005706_add_validate_course_number_to_event_kinds.rb

```diff
@@ -0,0 +1,10 @@
+#  Copyright (c) 2024, Pfadibewegung Schweiz. This file is part of
+#  hitobito_pbs and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class AddValidateCourseNumberToEventKinds < ActiveRecord::Migration[6.1]
+  def change
+    add_column :event_kinds, :validate_course_number, :boolean, default: false, null: false
```

Comment:
```suggestion
    add_column :event_kinds, :validate_course_number, :boolean, default: false, null: false
    
    Event::Kind.reset_column_information
```

Dies sollte meines Erachtens nach dem CI helfen die Spalte während der Specs zu finden